modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/services/GlobalL0Service.scala

```diff
@@ -180,16 +182,23 @@ object GlobalL0Service {
             )
         }
 
+      private def calculateLastOrdinal(nextOrdinal: SnapshotOrdinal, latestOrdinal: SnapshotOrdinal): SnapshotOrdinal =
+        SnapshotOrdinal.unsafeApply(
```

Comment:
are we 100% sure it will always satisfy `SnapshotOrdinal` refinement?

Reply:
@marcinwadon I believe this should always hold: `nextOrdinal < calculatedResult <= latestOrdinal`

Reply:
do I understand you correctly that the `nextOrdinal` is always lower than the `latestOrdinal`?

Reply:
If not then the result is `latestOrdinal`.